src/Common/TableEntity.tsx

```diff
@@ -143,7 +143,6 @@ export const TableEntity: FunctionComponent<TableEntityProps> = ({
                 src={EditIcon}
                 alt="editEntity"
                 onClick={onEditEntity}
-                tabIndex={0}
```

Comment:
You should remove the `tabIndex` from line 140 instead. If you remove this one, then pressing enter while the button is selected won't do anything.